api/v1alpha1/databasecluster_types.go

```diff
@@ -60,11 +60,18 @@ const (
 	// AppStateNew represents a newly created cluster that has not yet been reconciled.
 	AppStateNew AppState = ""
 
-	// ExposeTypeInternal is an internal expose type.
+	// Deprecated: ExposeTypeInternal is an internal expose type. Please use ExposeTypeClusterIP instead
 	ExposeTypeInternal ExposeType = "internal"
-	// ExposeTypeExternal is an external expose type.
+	// Deprecated: ExposeTypeExternal is an external expose type. Please use ExposeTypeLoadBalancer instead
```

Comment:
Can we emit a warning from the validation webhook when a user uses the deprecated types? This will make it more visible to the end users